File: V_1/TravkingApplicationAPI/DTO/AddSubTask.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace TravkingApplicationAPI.DTO
{
    public class AddSubTask
    {
    public string Title { get; set; }
    public string Description { get; set; }
    public int Status { get; set; }
    public int TaskId { get; set; }//Fk to Task

    public IFormFile? FileUploadTaskFileUpload{get;set;}

    public byte[]? FileUploadTaskPdf { get; set; }

    public string FileName{get;set;}
    }
}

File: V_1/TrackigApplicationFrontend/src/app/Models/add-sub-task.ts
export class AddSubTask {

  Title!: string;
  Description!: string;
  TaskId!: number[];
  FileUploadTaskFileUpload?: File|null; // Assuming this is for file uploads in Angular
  FileUploadTaskPdf?: Uint8Array|string|null;
  TestCases!:string;
  isProctored!:boolean;
}
